frontend/src/components/message.module.css

```diff
@@ -33,3 +33,8 @@
   line-height: 22px;
   font-size: 16px;
 }
+
+.messageStyle.single-line {
```

Comment:
I think if you remove the `align-items: center;` from .`container` class then you don't need this single-line class or the code changes in message.tsx?